from errors import MyError

class NotFound(MyError):
    pass